packages/devextreme/js/events/click.js

```diff
@@ -83,12 +84,9 @@ const ClickEmitter = Emitter.inherit({
         const getTarget = function(e) {
             const originalEvent = e.originalEvent;
 
-            if(originalEvent?.target?.shadowRoot) {
-                const path = originalEvent.path ?? originalEvent.composedPath?.();
-                return $(path[0]);
-            }
+            const target = originalEvent ? getEventTarget(originalEvent) : e.target;
```

Comment:
cannot we move this check for `originalEvent` into `getEventTarget` to reduce copypaste? I see that now we write similar code in all 3 places where we call it

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/27189/commits/a61f0524d90019a7baca359bd7062f2d419c1f91)